discovery-provider/scripts/start.sh

```diff
@@ -112,11 +112,13 @@ fi
 
 
 # start es-indexer
-if [[ "$audius_elasticsearch_url" ]] && [[ "$audius_elasticsearch_run_indexer" ]]; then
-  # npm run catchup creates triggers + populate indexes - this blocks server / celery start
-  # npm start gets backgrounded and goes into listen mode
-  cd es-indexer && npm i && npm run build && npm run catchup && npm start &
-fi
+(
+    if [[ "$audius_elasticsearch_url" ]] && [[ "$audius_elasticsearch_run_indexer" ]]; then
+    # npm run catchup creates triggers + populate indexes - this blocks server / celery start
+    # npm start gets backgrounded and goes into listen mode
+    cd es-indexer && npm run catchup && npm start &
```

Comment:
If we're not doing the `cd` outside of the if statement, I'd move the parens to be around line 119 specifically.

Reply:
Oh sorry... github failed to show these comments and I pressed merge... 
Do you want to apply these suggestions to #3245 since you're in here?
If not I can do as a follow on PR

Reply:
Yup! No worries. I'll handle that there :)